modules/Makefile.kubernetes

```diff
@@ -10,9 +10,10 @@ KUBERNETES_RESOURCE_VALIDATION ?= configmap
 export KUBERNETES_APP
 
 # Kubectl specific settings
+SSH_SOCK ?= ~/tmp/kubectl-$(CLUSTER_BASTION)
```

Comment:
I suggest using `/tmp/...` instead of under home. That way, the files get cleaned up automatically